tests/testsuite/future_incompat_report.rs

```diff
@@ -189,17 +189,17 @@ fn test_multi_crate() {
         p.cargo(command).arg("--future-incompat-report")
             .env("RUSTFLAGS", "-Zfuture-incompat-test")
             .with_stderr_contains("warning: the following packages contain code that will be rejected by a future version of Rust: first-dep v0.0.1, second-dep v0.0.2")
-            .with_stderr_contains("  - first-dep:0.0.1")
-            .with_stderr_contains("  - second-dep:0.0.2")
+            .with_stderr_contains("  - first-dep@0.0.1")
+            .with_stderr_contains("  - second-dep@0.0.2")
             .run();
 
-        p.cargo("report future-incompatibilities").arg("--package").arg("first-dep:0.0.1")
+        p.cargo("report future-incompatibilities").arg("--package").arg("first-dep@0.0.1")
```

Comment:
This test failed with `:` until I updated it because the package id is being written to the report.